releases/release-1.16/exceptions.yaml

```diff
@@ -0,0 +1,22 @@
+# Exceptions to Code Freeze requested in 1.16
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: Lachlan Evenson ([@lachie83](https://github.com/lachie83))
+
+- name: add status code and content check for HTTP Probe
```

Comment:
Nittest of Nits: Add should be capitalized to match the Issue title.

Reply:
that is nit picky... just rebased.